react/components/Collapsible/README.md

```diff
@@ -49,16 +31,15 @@ initialState = { isOpen: false }
         </span>
       }
       onClick={e => setState({ isOpen: e.target.isOpen })}
-      isOpen={state.isOpen}
-      >
-        <div className="mt4">
-          <div className="mb3">
-            <Input label="Surname" />
-          </div>
-          <div className="mb3">
-            <Input label="Nickname" />
-          </div>
+      isOpen={state.isOpen}>
```

Comment:
```suggestion
      isOpen={state.isOpen}
    >
```

Reply:
Strange, why wouldn't my Prettier do this by itself?

Reply:
because it doesn't look at the .md files